shesha-reactjs/src/designer-components/autocomplete/settingsForm.json

```diff
@@ -792,21 +792,37 @@
             "tablet": {},
             "mobile": {}
           },
+          {
+            "id": "7e4876d8-1ed4-4576-9f5b-46cfd77d8643",
+            "type": "checkbox",
+            "propertyName": "hideBorder",
```

Comment:
1. Property name doesn't match to the label and purpose of the setting
2. If you add new setting it should be properly initialized in the migrations